packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -335,6 +335,31 @@ const EDiscovery = WebexPlugin.extend({
       resource: 'reports/clientconfig',
       timeout: options.timeoutMs
     });
+  },
+
+  @waitForValue('@')
+  /**
+   * Submits an audit event to the eDiscovery service for admin use. Only expected to be used for
+   * the getContentSummary API
+   * @param {UUID} reportId - Id of the report to send an audit log for
+   * @param {Object} options - optional parameters for this method
+   * @param {number} options.timeoutMs - connection timeout in milliseconds, defaults to 30s
+   * @returns {Promise<Config>} Http response containing the config object
+   */
+  postAuditLog(reportId, options) {
+    if (!reportId) {
+      throw Error('Undefined parameter');
```

Comment:
Can you throw a more specific error message so it's more helpful for a developer trying to debug? 
```suggestion
      throw Error('Undefined reportId. Cannot audit report.');
```

Feel free to write something else, it's just a suggestion